winui/Rating/Clear-rating.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Clear rating value in WinUI Rating control | Syncfusion
+description: Learn all about how to enable the clear enable feature in the Rating control the user can remove their rating or not.
+platform: WinUI
```

Comment:
@Priyanka-Vijayasankar Learn all about how to clear the rating value in the WinUI Rating (SfRating) control.

Reply:
Updated the above changes.